src/SmartComponents/Inventory/applications/vulnerabilities/DataMapper.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { SecurityIcon } from '@patternfly/react-icons';
+
+const colorToImpact = {
+    High: '#ec7a08',
```

Comment:
should these be set as pf-global colors as well?

Reply:
Yeah, I just copied over those strings, but `var(--pf-global--danger-color--100)` would suit it much better.